Lib/opcode.py

```diff
@@ -31,28 +31,47 @@
 hascompare = []
 hasfree = []
 hasnargs = [] # unused
+assembler = []
 
 opmap = {}
 opname = ['<%r>' % (op,) for op in range(256)]
+virtual = {}
+
+RELATIVE = 0x1
+ABSOLUTE = 0x2
```

Comment:
Aren't we about to remove all absolute jumps?

Reply:
Yes, and then this flag will not be needed.